src/main/java/org/geppetto/model/neuroml/modelInterpreterUtils/PopulateContinuousProjectionTypes.java

```diff
@@ -77,10 +78,10 @@ public Type resolveProjectionImportType(Component projection, ImportType importT
 	 * @throws ModelInterpreterException
 	 */
 	private Variable extractConnection(Component projectionChild, ArrayType prePopulationType, Variable prePopulationVariable, ArrayType postPopulationType, Variable postPopulationVariable)
-			throws ModelInterpreterException
+            throws ModelInterpreterException, GeppettoVisitingException
 	{
 		ConnectionType connectionType = (ConnectionType) populateTypes.getTypeFactory().getSuperType(ResourcesDomainType.CONNECTION);
-
+                connectionType.getSuperType().add(this.geppettoModelAccess.getType(TypesPackage.Literals.CONNECTION_TYPE));
```

Comment:
@mattearnshaw Isn't there only one NeuroML type for Connection? I wasnt expecting the code to add the superType in 3 different places